command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
I think the purpose is to say if you have a compliant CLI it must adhere... But you could also have a different type of runtime interface such as a restful web interface.  We would just have to write up a spec for that if we wanted interop over such.